hardinfo.1

```diff
@@ -0,0 +1,57 @@
+.\" Manpage for hardinfo.
+.\" https://github.com/lpereira/hardinfo/.
+.TH HARDINFO "1" "April 2017" "0.5" "User Commands"
+.SH NAME
+hardinfo \- a system profiler and benchmark for Linux systems
+.SH USAGE
+hardinfo [OPTION...]
+.SH DESCRIPTION
+HardInfo is a system profiler and benchmark for Linux systems.
+
+HardInfo can gather information about your operating system 
+and detected hardware by them, perform benchmarks, and then 
+generate printable reports either in HTML or in plain text formats.
+
+
+Features include:
+ *\  System information, like kernel, version, desktop, users, network
+ *\  Hardware information about overall detected devices
+ *\  Report generation (in either HTML or plain text)
+ *\  Benchmark tests to see machine capabilities
+ *\  Benchmarks guides to compared machine featured capabilites
+
+.SH OPTIONS
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+Show help options
+.TP
+\fB\-r\fR, \fB\-\-generate\-report\fR
+creates a report and prints to standard output
+.TP
+\fB\-f\fR, \fB\-\-report\-format\fR
+chooses a report format (text, html)
+.TP
+\fB\-b\fR, \fB\-\-run\-benchmark\fR
+run benchmark; requires benchmark.so to be loaded
+.TP
+\fB\-l\fR, \fB\-\-list\-modules\fR
+lists modules
+.TP
+\fB\-m\fR, \fB\-\-load\-module\fR
+specify module to load
+.TP
+\fB\-a\fR, \fB\-\-autoload\-deps\fR
+automatically load module dependencies
+.TP
+\fB\-v\fR, \fB\-\-version\fR
+shows program version and quit
+.SH AUTHORS/CONTRIBUTOR
+Hardinfo its currently mantained by \fBLeandro A. F. Pereira\fR \fI<leandro@hardinfo.org>\fR with many contributions 
```

Comment:
"probably", you are very modest or dont want to be honest!

it should!, was a mistake by me!

now done!